Add tobira series endpoint to remove paths

owi92 opened the pull request:
This will allow removing ("unmounting") a series in Tobira.
The necessary endpoints in Tobira itself were already existing and also called by this endpoint in Opencast.
This commit just makes some form params optional.


### How to test this patch
<sub>The following is a bunch of information that is useful to test this, but in the interest of time I'd suggest that this should be tested by @Arnei, since he has done so before and won't need the extra preparation.</sub>
If you want to test this and haven't done any Tobira testing before (though that is probably too much work for this small PR for anyone that hasn't tested these Tobira endpoints before):
- You'll need at least Tobira 2.11 and configure it to talk to your Opencast with JWT setup (see our docs for reference: https://elan-ev.github.io/tobira/setup/auth/jwt)
- The relevant endpoint is `updateSeriesTobiraPath` and can be tested via `{your-opencast}/docs.html?path=/admin-ng/series#updateSeriesTobiraPath-10`
- In order to use that endpoint for series unmounting:
  -  you'll need the `seriesId` (Opencast ID) of a mounted series in Tobira
  - The page must not have any other content that the series
  - `pathComponents` must be empty
  - `currentPath` needs to be the path of that series in Tobira (i.e. "/page/my_series")
  - `targetPath` must be empty

<!--
If your pull request is targeting the legacy (second oldest release branch), explain why this patch is so important that
it needs to go into the legacy and can not go just into stable or develop.
-->

<!--
Explain how to test this patch. If this requires a particular set-up or configuration, explain that as well and provide
the necessary configuration if possible. The easier it is for others to test the patch, the faster this will get
reviewed and merged.
-->


### Your pull request should…

* [ ] have a concise title
* [ ] [close an accompanying issue](https://docs.opencast.org/develop/developer/#participate/development-process/#automatically-closing-issues-when-a-pr-is-merged) if one exists
* [ ] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] include migration scripts and documentation, if appropriate
* [ ] pass automated tests
* [ ] have a clean commit history
* [ ] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)
* [ ] explain why it needs to be merged into the legacy branch, if it is targeting the legacy branch

owi92 committed:
Make Tobira series endpoint more flexible

This will allow the `updateSeriesTobiraPath` endpoint to
also remove the series from a given path in Tobira by making
some form parameters optional.

gregorydlogan commented:
As requested in the body ;)

Arnei reviewed (changes requested):
Works. I would prefer to have separate endpoints for updating and deleting, to make the whole thing a little more CRUD (in line with the other endpoints). But if that does not make sense because of how Tobira handles things, then so be it.

owi92 committed:
Split Tobira mounting endpoint

This used the mount endpoint for mounting as well
as unmounting, which made the thing more complex
than it needed to be. Splitting this means less
optional parameters and with that, less failure
potential. Note that the `path` parameter for
unmounting needs to url encoded, which is not
an issue for the admin ui, but its kinda weird
for the testing form.
